### PR TITLE
#1351 Fix visibility

### PR DESCRIPTION
**Pull request by mximp**
#1351 Fix visibility

### CODE REVIEW

**Comment by mximp** [comment]
@volodya-lombrozo please check

File: eo-maven-plugin/src/main/java/org/eolang/maven/Home.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
  *  We also need to add new unit test
  */
 @SuppressWarnings("PMD.TooManyMethods")
-public class Home {
+class Home {
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/main/java/org/eolang/maven/Home.java:51]: can we add  a `final` modifier here?

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
Look good for me. Only one suggestion

File: eo-maven-plugin/src/main/java/org/eolang/maven/Home.java (hunk 1 above)
Comment (mximp) [reply eo-maven-plugin/src/main/java/org/eolang/maven/Home.java]: @volodya-lombrozo corrected. Thanks.

**Comment by mximp** [comment]
@volodya-lombrozo please re-check

**Review by volodya-lombrozo — APPROVED** [review]
lgtm

**Comment by mximp** [comment]
@yegor256 please merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**